src/libs/HttpUtils.js

```diff
@@ -21,6 +21,9 @@ Onyx.connect({
 // We use the AbortController API to terminate pending request in `cancelPendingRequests`
 let cancellationController = new AbortController();
 
+// To terminate pending ReconnectApp requests https://github.com/Expensify/App/issues/15627
```

Comment:
Please do not leave links to issues in the code. If someone needs to figure out where and why this was added they can use git blame.